backend/src/core/utils/validate-domain.ts

```diff
@@ -45,7 +62,7 @@ const validateDomain = async (
     )
   } else {
     logger.info({
-      message: 'Domains whitelisted',
+      message: 'Domains whitelisted in env var',
       domainsToWhitelist,
       action: 'validateDomain',
```

Comment:
In the end I didn't remove this log since presumably the env var should be quite short